publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceService.java

```diff
@@ -158,16 +165,16 @@ public void updateOwner(String identifier, UserInstance oldOwner, UserInstance n
             .collect(Collectors.toList());
     }
 
-    public Publication publishPublication(Publication dto)
-        throws InvalidPublicationException, NotFoundException {
-        return publishResource(Resource.fromPublication(dto)).toPublication();
+    public void publishPublication(UserInstance userInstance, SortableIdentifier resourceIdentifer)
```

Comment:
Cannot return the updated publication anymore because we update the resource and the doi-request at the same time in one transaction and dynamo does not return the updated values.